http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -41,6 +45,14 @@ private FeignClients() {}
      * com.fasterxml.jackson.datatype.guava.GuavaModule} and {@link com.fasterxml.jackson.datatype.jdk7.Jdk7Module}.
      */
     public static FeignClientFactory standard() {
+        return standard(DEFAULT_TIMEOUT_OPTIONS);
+    }
+
+    /**
+     * Provides a {@link FeignClientFactory} with an {@link ObjectMapper} configured with {@link
+     * com.fasterxml.jackson.datatype.guava.GuavaModule} and {@link com.fasterxml.jackson.datatype.jdk7.Jdk7Module}.
+     */
+    public static FeignClientFactory standard(Request.Options timeoutOptions) {
```

Comment:
Ok yes many things can be improved in the future, but this is specifically
so Foundry can stop copy/pasting the method with timeoutOptions and simply
use http-remoting. Are there objections to this commit as-is?